wiki/en/Running-a-Server.md

```diff
@@ -193,37 +186,13 @@ Now you can run these with the `systemctl` command, for example:
 
 You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
 
-### Making a Server status page
```

Comment:
Moved to Hints and Tips?

Reply:
No. I think this should no longer be documented - especially as we have JSON-RPC around the corner.

Reply:
JSON-RPC doesn't solve the problem this does.  This spits out HTML that can be used without further thought by anyone.  JSON-RPC needs a programmer to come along and write a program to send the request and parse the output and generate the HTML.

Reply:
Another one for T&T then?

Reply:
Ok. Moved.